package/gluon-config-mode-contact-info/luasrc/lib/gluon/config-mode/wizard/0500-contact-info.lua

```diff
@@ -19,6 +19,20 @@ function M.section(form)
   o.rmempty = not ((site.config_mode or {}).owner or {}).obligatory
   o.datatype = "string"
   o.description = i18n.translate("e.g. E-mail or phone number")
+  o.validate = function(self, val)
+    if val == nil then
+      self.section.error = {
+        [1] = {
```

Comment:
The `[1] = ` is redundant, as you assign a single table element, it will have the key `1` by default.